architecture/additional_concepts/storage.adoc

```diff
@@ -578,3 +578,175 @@ spec:
         claimName: myclaim
 
 ----
+
+[[blocl-volume-support]]
+
+== Block Volume Support
+[IMPORTANT]
+====
+Block Volume Support is a Technology Preview feature and it is only available for manually provisioned persistent volumes.
+ifdef::openshift-enterprise[]
+Technology Preview features are not supported with Red Hat production service
+level agreements (SLAs), might not be functionally complete, and Red Hat does
+not recommend to use them for production. These features provide early access to
+upcoming product features, enabling customers to test functionality and provide
+feedback during the development process.
+
+For more information on Red Hat Technology Preview features support scope, see
+https://access.redhat.com/support/offerings/techpreview/.
+endif::[]
+====
+
+You can statically provisioning Raw Block Volumes by including some new API fields in your PV and PVC specs.
+
+Example Persistent Volume:
+[source, yaml]
+----
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: block-pv
+spec:
+  capacity:
+    storage: 10Gi
+  accessModes:
+    - ReadWriteOnce
+  volumeMode: Block <1>
+  persistentVolumeReclaimPolicy: Retain
+  fc:
+    targetWWNs: ["50060e801049cfd1"]
+    lun: 0
+    readOnly: false
+----
+<1> volumeMode field indicating that this PV is a raw block volume
+
+Example Persistent Volume Claim:
+[source, yaml]
+----
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: block-pvc
+spec:
+  accessModes:
+    - ReadWriteOnce
+  volumeMode: Block <1>
+  resources:
+    requests:
+      storage: 10Gi
+----
+<1> VolumeMode field indicating that a raw block Persistent Volume is requested.
+
```

Comment:
`VolumeMode` field indicating that a raw block persistent volume is requested.

Reply:
@screeley44 lets keep the casing for volumeMode consistent to match the API